### PR TITLE
#1-4Task. Mikhail Pyshnik

### PR DESCRIPTION
**Pull request by MikhailPyshnik**
https://travis-ci.org/github/MikhailPyshnik/js-assignments/builds/704021352

### CODE REVIEW

File: task/04-arrays-tasks.js [hunk 1]
```diff
@@ -415,7 +440,24 @@ function toStringList(arr) {
...
  */
 function sortCitiesArray(arr) {
-   throw new Error('Not implemented');
+   return arr.sort(sortByRule);
+
+   function sortByRule(firstElement, secondElement)
+   {
+      if (firstElement['country'] == secondElement['country'])
+      {
+         return RuleOfSort(firstElement['city'], secondElement['city']);
```
Comment (AlexMokin) [inline task/04-arrays-tasks.js:449]: method should contain a verb in its name

**Comment by AlexMokin** [comment]
carefully review the comments. Task is accepted

File: task/04-arrays-tasks.js (hunk 1 above)
Comment (MikhailPyshnik) [reply task/04-arrays-tasks.js:449]: Hi Alex! I will correct all the comments, but later.

Comment (MikhailPyshnik) [reply task/04-arrays-tasks.js:449]: Hi Alex! Here is correct PR : https://github.com/AisBrestEDU/js-assignments/pull/149 . Notes fixed.

**Pull request closed without merging**